recipes/qt-advanced-docking-system/all/conanfile.py

```diff
@@ -51,6 +51,10 @@ def layout(self):
     def requirements(self):
         self.requires("qt/[>=5.15 <7]", transitive_headers=True)
 
+    def build_requirements(self):
+        self.tool_requires("qt/<host_version>")
```

Comment:
Please don't... it's a complete waste of build time, since the official .cmake modules will always use the executables they find in the host-context package folder.